<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductWindow</class>
 <widget class="QMainWindow" name="AddProductWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>添加商品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>287</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="bookButton">
       <property name="text">
        <string>书籍</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="foodButton">
       <property name="text">
        <string>食品</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="clothButton">
       <property name="text">
        <string>服装</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="bookFeature">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>版次</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="foodFeature">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="placeholderText">
     <string>保质期(days)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="clothFeature">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>尺码(号数)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>166</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>商家编号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="shopID">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>商品名称</string>
         </property>
         <property name="buddy">
          <cstring>name</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>商品描述</string>
         </property>
         <property name="buddy">
          <cstring>description</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="description"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>商品价格</string>
         </property>
         <property name="buddy">
          <cstring>price</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="price"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>商品库存</string>
         </property>
         <property name="buddy">
          <cstring>stock</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stock"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>okButton</zorder>
   <zorder>label_5</zorder>
   <zorder>cancelButton</zorder>
   <zorder>clearButton</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>foodFeature</zorder>
   <zorder>clothFeature</zorder>
   <zorder>bookFeature</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bookButton</sender>
   <signal>clicked()</signal>
   <receiver>bookFeature</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>foodButton</sender>
   <signal>clicked()</signal>
   <receiver>foodFeature</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clothButton</sender>
   <signal>clicked()</signal>
   <receiver>clothFeature</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookButton</sender>
   <signal>clicked()</signal>
   <receiver>clothFeature</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookButton</sender>
   <signal>clicked()</signal>
   <receiver>foodFeature</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clothButton</sender>
   <signal>clicked()</signal>
   <receiver>bookFeature</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clothButton</sender>
   <signal>clicked()</signal>
   <receiver>foodFeature</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>foodButton</sender>
   <signal>clicked()</signal>
   <receiver>bookFeature</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>foodButton</sender>
   <signal>clicked()</signal>
   <receiver>clothFeature</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
